frontend/src/pages/phone.page.tsx

```diff
@@ -42,7 +42,7 @@ const Phone: NextPage = () => {
   }, [isInOnboarding, relayNumberData]);
 
   // check if phone flag is active - return to premium page if not.
-  if (isFlagActive(runtimeData.data, "phones")) {
+  if (!isFlagActive(runtimeData.data, "phones")) {
     router.push("/premium");
   }
```

Comment:
Ah, build is failing, due to the flag not being active when prerendering, when there's no route yet:

```suggestion
  useEffect(() => {
    if (!isFlagActive(runtimeData.data, "phones")) {
      router.push("/premium");
    }
  }, [runtimeData.data]);
```